src/chi/foundations/typography/typography.scss

```diff
@@ -35,51 +35,51 @@ a {
 }
 
 h1,
-.chi-h1 {
-  font-size: 3rem;
-  font-weight: 700;
-  line-height: 4rem;
-  margin: 2rem 0;
+.-text--h1 {
```

Comment:
@jllr @janibolkvadze  !important should only be defined on the text utility class, not the heading elements

Reply:
Heading tag styles are default rules, text utility styles are customization rules. Would be clearer if we leave default heading styles in typography and move text utility styles to utilities/text.